examples/nlp_finetuning/renate_config.py

```diff
@@ -13,26 +12,24 @@
 from renate.models.renate_module import RenateWrapper
 
 
-def model_fn(model_state_url: Optional[Union[Path, str]] = None) -> RenateModule:
+def model_fn(model_state_url: Optional[str] = None) -> RenateModule:
```

Comment:
If we're disallowing path objects here, should we also do that in the other examples?

Reply:
the use of complex types as Path is no longer supported for our *_fn functions since 0.2.0. I can check for more occurrences but I should have removed most of them before

Reply:
I see occurrences of `Path` in most example config files. If you don't want to remove then now, we should create an issue to do it later.

Reply:
I've updated the remaining config files as well as the documentation.

Reply:
Nice, thanks!